controllers/articlesEndPointController.js

```diff
@@ -0,0 +1,22 @@
+const { getArticleById } = require("../models/articlesEndPointModel");
+
+const getSinlgeArticle = (req, res, next) => {
+  const articleID = req.params.article_id;
+
+  if (isNaN(articleID)) {
+    return res.status(400).send({ msg: "Bad Request" });
+  }
+  getArticleById(articleID)
+    .then((article) => {
+      if (!article) {
+        return res.status(404).send({ msg: "Article not found" });
```

Comment:
I like this logic you've got here, however it might be better to have this logic in the Model and use `Promise.reject` instead, which then gives you an error you can handle in the catch

Reply:
done